### PR TITLE
Add trailing newline character in java home command

### PR DESCRIPTION
**Pull request by irakliy01**
This PR fixes [965](https://github.com/sdkman/sdkman-cli/issues/965) bug.

Need to update config file loaded on sdk install

### CODE REVIEW

**Comment by irakliy01** [comment]
https://github.com/sdkman/sdkman-cli/pull/990

File: src/main/bash/sdkman-home.sh [hunk 1]
```diff
@@ -35,5 +35,10 @@ function __sdk_home() {
 		return 1
 	fi
 
-	echo -n "${SDKMAN_CANDIDATES_DIR}/${candidate}/${version}"
+		
+	if [[ "$sdkman_home_newline" == "false" ]]; then
```
Comment (leagris) [inline src/main/bash/sdkman-home.sh:39]: Make the test use POSIX-shell grammar
```sh
	if [ "$sdkman_home_newline" = 'false' ]; then
```

File: src/main/bash/sdkman-home.sh [hunk 2]
```diff
@@ -35,5 +35,10 @@ function __sdk_home() {
...
+    	echo -n "${SDKMAN_CANDIDATES_DIR}/${candidate}/${version}"
```
Comment (leagris) [inline src/main/bash/sdkman-home.sh:40]: In POSIX sh, echo flags are undefined. [SC3037](https://github.com/koalaman/shellcheck/wiki/SC3037)
Replace by:
```sh
printf %s "${SDKMAN_CANDIDATES_DIR}/${candidate}/${version}"
```

File: src/main/bash/sdkman-home.sh [hunk 3]
```diff
@@ -35,5 +35,10 @@ function __sdk_home() {
...
+    else
+    	echo "${SDKMAN_CANDIDATES_DIR}/${candidate}/${version}"
+	fi
```
Comment (leagris) [inline src/main/bash/sdkman-home.sh:43]: This whole test can be replaced by POSIX-shell grammar shorter form:
```sh
printf %s "${SDKMAN_CANDIDATES_DIR}/${candidate}/${version}"
[ "$sdkman_home_newline" = 'false' ] || echo
```

**Review by leagris — COMMENTED** [review]
The SDKMan installer care to be compatible with POSIX-shell grammar in a way, but then the whole project code seems to just care about Bash. Ill check this overall. Issue not specific to this PR.

**Pull request closed without merging**